### ISSUE TITLE
Setup Spina from a repository 

### ISSUE THREAD

**Issue by andywenk** [denkGroot/Spina#294]
Hey there,

first of all thanks for Spina. It looks and feels extremely cool. 

**Description**

I have a git repository where I already made changes. I am wondering what the process is, when someone wants to work on it from the repo. 

**Steps**

First she will clone the repo. Ok. I then thought I simply create and admin in the seed.rb file. But I realised that there are some bits missing. So obviously one has to use the installer. But then, I would overwrite some basic stuff if I am not careful. Well, this could be placed in the README file.

**Other env's**

The next question is then, how I would handle this, when deploying to a staging environment.  Do I have to use the Spina installer again? 

**production**

And last but not least I saw, that in the installer script, some things are not being run if the Rails.env is production. I suspect that my problems would be fixed, if I would run the local env in production right?

**possible solution**

One thing would be to use sth. like https://github.com/rroblak/seed_dump, create a basic setup and run this instead of the installer after cloning.

What is the proposed way here?

Thanks a lot and keep on with the great work.

All the best

Andy

**Comment by Bramjetten** [comment]
Hey Andy!

Thanks for your support!

I'm not entirely sure what you mean by setting up Spina from a repository. Do you mean from an existing Rails app? 

The install generator can be used both locally to bootstrap Spina and in production. It automatically skips the steps that shouldn't be necessary (like copying migration and template files from the gem).

You should just run the installer in your regular development environment, it'll copy some required files in your Rails app, run the migrations and seed some data.

We're currently hard at work developing our new website which includes proper documentation on how to get started using Spina. 

Bram

**Comment by andywenk** [comment]
Hey Bram,

thanks a lot for your answer. I meant that I did setup Spina and pushed it to our GitLab instance as a new repo. Then other developers will clone it and work further on it. 

I played around further yesterday and at the end everything worked. I used the generator also on the staging servers and that's just fine. So thanks for your answer. 

I am looking forward to see the documentation. When finding some "things to fix or improve", I will send a PR.

All the best

Andy
